tools/cythonize.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python
+""" cythonize
+
+Cythonize pyx files into C files as needed.
+
+Usage: cythonize [root_dir]
+
+Default [root_dir] is 'sklearn'.
+
+Checks pyx files to see if they have been changed relative to their
+corresponding C files.  If they have, then runs cython on these files to
+recreate the C files.
+
+The script thinks that the pyx files have changed relative to the C files
```

Comment:
Maybe "the script detects changes in the pyx files using checksums [or hashes] stored in a database file" ?